test/configCases/plugins/define-plugin/index.js

```diff
@@ -1,89 +1,104 @@
-it("should define FALSE", function() {
-	FALSE.should.be.eql(false);
-	(typeof TRUE).should.be.eql("boolean");
-	var x = require(FALSE ? "fail" : "./a");
-	var y = FALSE ? require("fail") : require("./a");
-});
+/* globals define, it, should */
+define("DefinePlugin", function() {
```

Comment:
It' describe actually, but not needed here, as the folder name is already used as suited name.